.github/workflows/documentation.yml

```diff
@@ -39,9 +39,12 @@ jobs:
       with:
         ref: gh-pages
         clean: false
-        
+
     - name: Move generated files where git can see them
       run: cp -rp docs/html/* .
+        
+    - name: Prevent generated docs dir from being committed and overwriting on the next run.
+      run: rm -Rf docs/html
 
     - name: Prevent generated docs dir from being committed and overwriting on the next run.
```

Comment:
Why and how did this step get duplicated?

Reply:
No idea. The only way that I can think of is during rebasing. At no time that I even touched that file. Did rebasing senses something different even human eyes see they are exactly the same? Any way, I will remove the repeat.